docs/src/assets/examples/array.raml

```diff
@@ -0,0 +1,58 @@
+#%RAML 1.0
+#baseUri: http://www.tshirt.com/api
```

Comment:
Didn't find any [build options](https://www.npmjs.com/package/api-console-builder#options) for keeping it more minimal. Can go through generated items manually and write a list of files/folders that can be deleted, but that seems painful, error-prone (maybe something gets deleted that leads to an error only way later) and wouldn't scale well with api-console version bumps. Though I expect upgrades on our side to be very rare.